content/en/blog/2023/steering-contribution-seat-results/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "2023 Istio Steering Contributor Seats"
+subtitle: Announcing the Istio Steering Committee Contributor Members, 2023.
+description: Announcing the Istio Steering Committee Contributor Members, 2023.
+publishdate: 2023-02-04
+attribution: "Faseela K, for the Istio Steering Committee"
+keywords: [istio,steering,governance,community,contributor]
+---
+
+The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 Contribution Seats, proportionally allocated based on corporate contributions to the project, and 4 elected Community Seats.
+
+Last year, we [elected the four members](/blog/2022/steering-election-results/) to the community seats. It's now time to announce the companies who fuel our growth by selecting the [Contribution Seat] members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).
```

Comment:
elected ~~the~~ four

Reply:
Do we want []'s around contribution seat? It seems we are missing a link?

Reply:
i think it was a merging mistake. fixed.